src/Test.FunctionalTests.BinSkim.Driver/BaselineTestData/Expected/clangcl.pe.cpp.codeview.exe.sarif

```diff
@@ -3,60 +3,995 @@
+            },
+            {
+              "id": "BA2026",
+              "name": "EnableMicrosoftCompilerSdlSwitch",
+              "fullDescription": {
+                "text": "/sdl enables a superset of the baseline security checks provided by /GS and overrides /GS-. By default, /sdl is off. /sdl- disables the additional security checks."
+              },
+              "help": {
+                "text": "/sdl enables a superset of the baseline security checks provided by /GS and overrides /GS-. By default, /sdl is off. /sdl- disables the additional security checks."
+              },
+              "messageStrings": {
+                "Pass": {
+                  "text": "'{0}' is a Windows PE that was compiled with recommended Security Development Lifecycle (SDL) checks. These checks change security-relevant warnings into errors, and set additional secure code-generation features."
+                },
+                "Warning": {
+                  "text": "'{0}' is a Windows PE that wasn't compiled with recommended Security Development Lifecycle (SDL) checks. As a result some critical compile-time and runtime checks may be disabled, increasing the possibility of an exploitable runtime issue. To resolve this problem, pass '/sdl' on the cl.exe command-line, set the 'SDL checks' property in the 'C/C++ -> General' Configuration property page, or explicitly set the 'SDLCheck' property in the project file (nested within a 'CLCompile' element) to 'true'."
+                },
+                "NotApplicable_InvalidMetadata": {
+                  "text": "'{0}' was not evaluated for check '{1}' as the analysis is not relevant based on observed metadata: {2}."
+                }
+              },
+              "helpUri": "https://github.com/microsoft/binskim/blob/main/docs/BinSkimRules.md#rule-BA2026EnableMicrosoftCompilerSdlSwitch"
             }
           ],
-          "executionSuccessful": false
+          "properties": {
+            "comments": "A security and correctness analyzer for portable executable and MSIL formats."
+          }
+        }
+      },
+      "invocations": [
+        {
+          "executionSuccessful": true
         }
       ],
       "artifacts": [
         {
           "location": {
-            "uri": "file:///Z:/src/Test.FunctionalTests.BinSkim.Driver/BaselineTestData/clangcl.pe.cpp.codeview.exe"
+            "uri": "file:///Z:/src/Test.FunctionalTests.BinSkim.Driver/BaselineTestData/clangcl.pe.cpp.codeview.exe",
+            "index": 0
           },
           "hashes": {
-            "md5": "F983F893F6C5181CFA6574B520143F07",
-            "sha-1": "AA8AC9104D7917243AE052942C5F40BA929415AF",
-            "sha-256": "AC5BDD260432A32F62756ED8CF6E4297DAFFF0E6BA01C9522CC956C6D894B5C2"
+            "md5": "3AA0BC84045DCF91EF2B136E0B741200",
+            "sha-1": "C5EBC1B2D5BCCF44D69A61C437A7701958B30E07",
+            "sha-256": "918D85AEEDEACCC6D9E9E64C851560900BDA0CB554148645FF0295E403A21979"
```

Comment:
note the hashes change. 
This was a bug from SDK.

my comment here:
[https://github.com/microsoft/binskim/pull/853#discussion_r1164644217](https://github.com/microsoft/binskim/pull/853#discussion_r1164644217)